pyblish_nuke/lib.py

```diff
@@ -278,15 +279,15 @@ def dock(window):
 
         dialog.deleteLater()
 
-        # if there are more than one main window, its a floating window
+        # If there are more than one main window, its a floating window
         if len(main_windows) > 1:
```

Comment:
Sure, I'll have a look at, although I'm pretty blind to this kind of stuff so I can't fix other commenting :)